lib/Twig/Node/Expression/EmptyCoalesce.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Twig\Node\Expression\EmptyCoalesceExpression;
+
+@trigger_error(sprintf('Using the "Twig_Node_Expression_EmptyCoalesce" class is deprecated since Twig version 2.7, use "Twig\Node\Expression\EmptyCoalesceExpression" instead.'), E_USER_DEPRECATED);
+
+class_exists('Twig\Node\Expression\EmptyCoalesceExpression');
+
+if (\false) {
+    class Twig_Node_Expression_EmptyCoalesce extends EmptyCoalesceExpression
```

Comment:
introducing the legacy class and deprecating it directly does not make sense.

Reply:
I'd be happy to remove this; would you like me to do that?

Reply:
I pushed a change to remove the pointless `Twig_Node_Expression_EmptyCoalesce` class entirely, which should resolve this.